### ISSUE TITLE
topology_raft_disabled.test_raft_upgrade fails

### ISSUE THREAD

**Issue by denesb** [scylladb/scylladb#13187]
I cannot determine the reason. Seen in promotion and CI lately, most recently here: https://jenkins.scylladb.com/job/releng/job/Scylla-CI/4949/

**Comment by kbr-scylla** [comment]
The failure happened during cluster setup before test
```
E               test.pylib.rest_client.HTTPError: HTTP error 500, uri: http+unix://api/cluster/before-test/test_recovery_after_majority_loss, params: None, json: None, body:
E               Exception when starting cluster ScyllaCluster(name: 6af78d1a-c29e-11ed-8429-125d80d9f0a3, running: ScyllaServer(135, 127.208.204.42, 1843206a-0999-4a23-9156-1868776e6b0c), stopped: ):
E               Failed to start server with ID = 137, IP = 127.208.204.35.
E               Check the log files:
E               /jenkins/workspace/releng/Scylla-CI/scylla/testlog/x86_64/test.py.debug-release-dev.log
E               /jenkins/workspace/releng/Scylla-CI/scylla/testlog/x86_64/release/scylla-137.log
```

In the logs
https://jenkins.scylladb.com/job/releng/job/Scylla-CI/4949/artifact/testlog/x86_64/release/scylla-137.log

we see
```
INFO  2023-03-14 21:28:42,229 [shard 0] cdc - Inserting new generation data at UUID 5e6a3a72-436a-4459-aece-bde4b38e997e
...
ERROR 2023-03-14 21:28:42,699 [shard 0] init - Startup failed: exceptions::unavailable_exception (Cannot achieve consistency level for cl ALL. Requires 1, alive 0)
```

the bootstrapping node was trying to insert CDC generation data in the `system_distributed_everywhere` keyspace, but for some reason it considered the other node as dead.

We can actually see that it marked the node as UP 2 ms after it tried to do the insert :(
```
INFO  2023-03-14 21:28:42,229 [shard 0] cdc - Inserting new generation data at UUID 5e6a3a72-436a-4459-aece-bde4b38e997e
INFO  2023-03-14 21:28:42,230 [shard 0] init - Shutting down group 0 service
INFO  2023-03-14 21:28:42,231 [shard 0] gossip - InetAddress 127.208.204.42 is now UP, status = NORMAL
```

This is sad. The two nodes talked already: the bootstrapping node had to contact the seed during shadow round. Why it takes so long for it to mark the node as UP, I don't understand.

Perhaps we should insert a "wait for nodes to be up" at some point.

**Comment by kbr-scylla** [comment]
Note that the failure happened in a suite where Raft is *disabled*.

**Comment by kbr-scylla** [comment]
Should be fixed by 0b19a614fa52980f6fac2ce2f8841631d5594e54. Closing. Can reopen if reproduces.
